haskell/src/Monocle/Prelude.hs

```diff
@@ -126,6 +126,7 @@ module Monocle.Prelude
     preview,
     at,
     set,
+    (?~),
```

Comment:
It doesn't seem used anymore, and I think we should avoid adding these lens operator.